pull-request.py

```diff
@@ -234,6 +234,20 @@ def find_pull_request(listing, source):
                 return entry
 
 
+def find_default_branch():
+    """Find default branch for a repo (only called if branch not provided)
+    """
+    response = requests.get(REPO_URL)
+
+    # Case 1: 404 might need a token
+    if response.status_code == 404:
+        response = requests.get(REPO_URL, headers=HEADERS)
+    if response.status_code != 200:
+        abort_if_fail(response, "Unable to retrieve default branch")
+
```

Comment:
Could  we put one more line here (before returning anything) that says something like "Found default branch X"? That way you can easily see in the action that the function was being used (and what it returned).

Reply:
Sure thing.  I was originally planning to do that, and then completely forgot about it when I wrote the actual code. :D

I used an f-string, but then as I was hitting return on the `git push`, I noticed the rest of the code uses the printf-style string replacements.  Want me to change it to match?

Reply:
I would be conservative and use print style string replacements.

Reply:
Ok - printf'd Line 249